src/d3a/d3a_core/export.py

```diff
@@ -132,6 +133,22 @@ def export(self, export_plots=True, power_flow=None):
     def data_to_csv(self, area, is_first):
         self._export_area_with_children(area, self.directory, is_first)
 
+    def area_tree_summary_to_json(self, data: Dict):
+        subdirectory = pathlib.Path(self.directory, "raw_data")
+        if not subdirectory.exists():
+            subdirectory.mkdir(exist_ok=True, parents=True)
+        json_file = os.path.join(self.directory, "area_tree_summary.json")
+        with open(json_file, 'w') as outfile:
+            json.dump(data, outfile, indent=2)
+
+    def raw_data_to_json(self, time_slot, data: Dict):
+        subdirectory = pathlib.Path(self.directory, "raw_data")
+        if not subdirectory.exists():
+            subdirectory.mkdir(exist_ok=True, parents=True)
```

Comment:
you could create this subdirectory before launching the simulation and also deleting it at the end (after exporting of course) Thanks!

Reply:
As far as I see `.exists()` already check if the subdirectory needs to be created or not, so why to replace with another check if it has to be created at start or not, considering the second point of deleting at the end, I would be skeptical to delete it at the end either as the integration tests uses it for testing the past markets stats after the simulation successfully finishes.

Reply:
Please think about performance. IMO we should not check every market slot whether the "raw_data" folder exists. We should just create it at the beginning and save 3 (or if you count area_tree_summary_to_json, too, 6) extra commands.
As for the deletion of the data: IMO there should be a global constant that (or you reuse D3A_TEST_RUN) that decides of keeping the rwa_data results. By default it should not keep the. Only for tests and debugging.

Reply:
Created raw data dir at the time of instantiating class `ExportAndPlot` as well as created an story to deal with the same pattern being found in old places like for CSV output: ExportAndPlot https://gridsingularity.atlassian.net/browse/D3ASIM-2834